src/baseline_automatic_pipeline_system/coreference.py

```diff
@@ -151,4 +168,31 @@ def some_arg_match(prop_mention1_info, prop_mention2_info, argument_clusters, ar
 
     else:
         return (matched_arguments / (len(prop_mention1_args) + len(
-            prop_mention2_args) - matched_arguments) >= arg_match_ratio)
\ No newline at end of file
+            prop_mention2_args) - matched_arguments) >= arg_match_ratio)
+
+
+def eval_coref_with_gold_mentions(gold_folder):
```

Comment:
I think it will be more efficient for our testing and debugging if the function would be "eval_coref_with_gold_graph" that take a single graph and return the coref evaluation (curr_entity_scores). the logic of iterating a folder, averaging the scores and printing, I would move to the "main", or to another function.